### PR TITLE
cleans up definitions in docstrings

### LINKED ISSUE #37: Clean through docstrings in scripts
**Describe the bug**
Docstrings in scripts (such as TODOs, Headframes) are being listed as description properties in the schemas. 

**Reproduce the problem**
1. print(Procedures.schema_json(indent=3)) and see property Procedure underneath Headframe
2. print(Subject.schema_json(indent=3)) and see properties' (ex: Sex, Background Strain) have description: "TODO"

**Screenshots**
Screen Shot 2022-11-01 at 2 45 21 PM
Screen Shot 2022-11-01 at 3 50 11 PM

### PR DESCRIPTION
**Pull request by dyf**
resolves #37

### CODE REVIEW

File: src/aind_data_schema/ephys/ephys_rig.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ class Laser(BaseModel):
 
 
 class Monitor(BaseModel):
-    """description of a physical display"""
+    """Description of a physical display"""
```
Comment (saskiad) [inline src/aind_data_schema/ephys/ephys_rig.py:99]: Description of a visual monitor

File: src/aind_data_schema/ephys/ephys_session.py [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
 
 class Institution(Enum):
-    """TODO"""
+    """Institution name"""
 
     AIND = "AIND"
     MSP = "MSP"
 
 
 class SessionType(Enum):
```
Comment (saskiad) [inline src/aind_data_schema/ephys/ephys_session.py:19]: I think this is a fine docstring, but this is another place where we need to get some team clarity (similar to project names)

File: src/aind_data_schema/ephys/ephys_session.py [hunk 3]
```diff
@@ -38,58 +38,58 @@ class ExpectedDataStream(Enum):
...
-    """description of 3d coordinates in MM"""
+    """Description of 3d coordinates in mm"""
 
     direction: Direction = Field(..., title="Direction")
     value: float = Field(..., title="Value (mm)", units="mm")
 
 
 class AnatomicalDirection(Enum):
-    """TODO: combine with imaging direction"""
+    """Anatomical diretion name (TODO: combine with imaging direction)"""
```
Comment (saskiad) [inline src/aind_data_schema/ephys/ephys_session.py:62]: typo in direction

File: src/aind_data_schema/ephys/ephys_session.py [hunk 4]
```diff
@@ -38,58 +38,58 @@ class ExpectedDataStream(Enum):
...
+    """Euler angle name"""
 
     XY = "XY"
     XZ = "XZ"
     YZ = "YZ"
 
 
 class ManipulatorAngles(BaseModel):
-    """angles with units"""
+    """Angles with units"""
```
Comment (saskiad) [inline src/aind_data_schema/ephys/ephys_session.py:85]: I don't understand what "with units" means here. We specify the units for this - as we do for other things.
Could this be "Angles of manipulator" or something like that?

File: src/aind_data_schema/imaging/acquisition.py [hunk 5]
```diff
@@ -68,7 +68,7 @@ class Laser(BaseModel):
 
 
 class Position(BaseModel):
-    """stage position"""
+    """Stage position information"""
```
Comment (saskiad) [inline src/aind_data_schema/imaging/acquisition.py:71]: is there a reason this is "XX information" rather that "Description of XX" ? I don't feel strongly, but the others (to the extent that I've noticed) are all "Description of ..."

File: src/aind_data_schema/imaging/instrument.py [hunk 6]
```diff
@@ -31,7 +31,7 @@ class MicroscopeManufacturer(Enum):
 
 
 class Ao(BaseModel):
-    """TODO"""
+    """Description of adaptive optics"""
```
Comment (saskiad) [inline src/aind_data_schema/imaging/instrument.py:34]: Is this really adaptive optics? I thought it was something different

File: src/aind_data_schema/imaging/instrument.py [hunk 7]
```diff
@@ -40,22 +40,22 @@ class Ao(BaseModel):
...
-    """TODO"""
+    """Description of a communication system"""
 
     hardware_name: str = Field(..., title="Controlled hardware device")
     com_port: str = Field(..., title="COM port")
 
 
 class CameraType(Enum):
-    """camera type list"""
+    """Camera type name"""
```
Comment (saskiad) [inline src/aind_data_schema/imaging/instrument.py:50]: Can it just be "Camera type" ? This to me reads more as a brand or model name - even though I know that's not what it is.

File: src/aind_data_schema/imaging/instrument.py [hunk 8]
```diff
@@ -138,17 +138,17 @@ class Coupling(Enum):
...
+    type: LightsourceType = Field(..., title="Lightsource Type")
     coupling: Coupling = Field(..., title="Coupling")
     wavelength: float = Field(..., title="Wavelength (nm)")
     max_power: float = Field(..., title=" Maximum power (mW)")
     filter_wheel_index: int = Field(..., title="Filter index")
 
 
 class Immersion(Enum):
-    """immersion media"""
+    """immersion media name"""
```
Comment (saskiad) [inline src/aind_data_schema/imaging/instrument.py:151]: Sentence case

File: src/aind_data_schema/processing.py [hunk 9]
```diff
@@ -10,7 +10,7 @@
 
 
 class ProcessName(Enum):
-    """list of processing labels"""
+    """Processing type labels"""
```
Comment (saskiad) [inline src/aind_data_schema/processing.py:13]: Data processing type labels?

**Review by saskiad — COMMENTED** [review]
A bunch of small comments - a couple need fixing, others are thoughts.

File: src/aind_data_schema/imaging/instrument.py (hunk 6 above)
Comment (dyf) [reply src/aind_data_schema/imaging/instrument.py:34]: this was a guess, I will un-guess

Comment (dyf) [reply src/aind_data_schema/imaging/instrument.py:34]: answer: analog output

**Review by saskiad — APPROVED** [review]

**Pull request merged**